apps/web/src/clients/api-client.ts

```diff
@@ -316,23 +316,21 @@ export class ApiClient {
       .catch(() => null)
   }
 
-  async getPayments(query: AdminPaymentListQuerystring) {
+  async getPayments(query: PaymentsQuerystring) {
     const searchQuery = getPaymentsFilterQuery(query)
-    return await this.http
-      .get(`payments?${searchQuery}`)
-      .json<AdminPaymentList>()
+    return await this.http.get(`payments?${searchQuery}`).json<Payments>()
   }
 
   async getAdminPaymentById(paymentId: string) {
     return await this.http
-      .get(`payments/${paymentId}/admin`)
-      .json<AdminPaymentBase>()
+      .get(`payments/${paymentId}?isAdmin=${true}`)
+      .json<Payment>()
   }
 
   async getPaymentsByBankAccountId(bankAccountId: string) {
     return await this.http
-      .get(`bank-accounts/${bankAccountId}/payments`)
-      .json<Payment[]>()
+      .get(`payments/bank-accounts/${bankAccountId}/payments`)
+      .json<ToPaymentBase[]>()
```

Comment:
Accidentally was using the `Payment` type, which includes additional database the database such as the `payerId`, but we just need to return a list of payments from Circle, some of which could not exist in the database.